script/zephyr_module.py

```diff
@@ -0,0 +1,313 @@
+#!/usr/bin/python
+
+# Copyright (c) 2019 STMicroelectronics.
+# This script will update the stm32cube module for Zephyr
+# from the latest version on https://github.com/STMicroelectronics
+# get the patch between initial version and latest version
+# apply this patch on the zephyr module
+# usage : 'python zephyr_module.py <stm32_serie>'
+
+import os
+import sys; sys.dont_write_bytecode = True    # do not generate pyc file
+import subprocess
+import time
+import make_module
+import shutil
+
+
+# execute with argument
+option_clean = False
+option_force = False
+module_serie = ''
+arg_names = ['command', 'a1', 'a2', 'a3']
+args = map(None, arg_names, sys.argv)
+args = {k: v for (k, v) in args}
+
+if args['a1']:
+    if 'stm32' in args['a1']:
+        module_serie = args['a1']
+    elif 'c' in args['a1']:
+        option_clean = True
+    elif 'f' in args['a1']:
+        option_force = True
+    else:
+        print 'unknown option'
+if args['a2']:
+    if 'c' in args['a2']:
+        option_clean = True
+    elif 'stm32' in args['a2']:
+        module_serie = args['a2']
+    elif 'f' in args['a2']:
+        option_force = True
+    elif 'f' in args['a2']:
+        option_force = True
```

Comment:
Not sure about this line (copy paste from previous line)

Reply:
fixed